lib/gears/shape.lua

```diff
@@ -67,7 +67,9 @@ function module.star(cr, width, height, n)
     n = n or 5
     local a = 2 * math.pi / n
 
+
```

Comment:
Now you have two empty lines here (both line 69 and line 70 are empty)

Reply:
I'm gonna remove it right away! :+1: